docs/use-existing-kafka.md

```diff
@@ -171,3 +171,38 @@ When using SSL to secure kafka communication, you must set `KAFKA_ENABLE_TLS`, a
 * `KAFKA_CACERTS` to check server certificate
 * `KAFKA_CERT` and `KAFKA_KEY` to check client certificate
 * `KAFKA_INSECURE` to skip TLS verfication
+```yaml
```

Comment:
would you mind adding some paragraph before doing what you are setting here? Something like "Below you can find an example of a Kafka controller deployment using TLS"